js/schedule.js

```diff
@@ -670,6 +688,44 @@ function Schedule(options) {
             }
         });
 
+        // toggle individual schedule blocks on header tap
+        schedule.$container.on('click', '.slider-control', function(e) {
+            var clicked = $(this);
+            var targetBlock = clicked.next('.page-block');
+            targetBlock.toggleClass('closed');
+            if (targetBlock.hasClass('closed')) {
```

Comment:
wondering if we should extract the following if-else into an animation helper function so we can reuse it for "toggle all?